get/getUserAvatar/index.js

```diff
@@ -1,29 +1,29 @@
-var AWS = require('aws-sdk');
+const AWS = require('aws-sdk');
 AWS.config.update({region: 'us-east-2'});
-var s3  = new AWS.S3();
-var srcBucket = "accedia-users-avatars";
+const ddb = new AWS.DynamoDB();    
 exports.handler = async (event) => {    
-    var ddb = new AWS.DynamoDB();    
-    var params = {
+    const params = {
         "TableName" : "Users",
-        "ProjectionExpression" : "username"
+        "ProjectionExpression" : "avatarUrl"
     };
-
+    let result;
     try{
-        var users = (await ddb.scan(params).promise()).Items;
-        
+        let users = (await ddb.scan(params).promise()).Items;
+        result = {
+            statusCode: 200,
+            body: []
+        }
         for(let i = 0; i < users.length; i++){
-            let data = await s3.getObject({
-                Bucket: srcBucket,
-                Key: users[i].username.S
-            }).promise();
-            users[i].avatar = data.Body.toString('base64');
+            result.body[i] = users[i].avatarUrl.S;
         }        
+        result.body = JSON.stringify(result.body);
     }catch(err){
-        users = err.message;
+        console.log(err);
+        result = {
+            body: "Internal server error!",
+            statusCode: 500 
```

Comment:
Change the status code to 400 and change the body message to "Bad input"
```suggestion
            statusCode: 400 
```